homework/index.js

```diff
@@ -30,13 +30,118 @@
     return elem;
   }
 
+  function setInnerText(repository, nthChild, text) {
+    document.querySelector('tbody').childNodes[nthChild].lastChild.innerText = text;
+  }
+
+  function presentRepositories(leftHandColumn, repository) {
+    const table = createAndAppend('table', leftHandColumn);
+    const tBody = createAndAppend('tbody', table);
+    const headsOfDetails = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    for (let i = 0; i < 4; i++) {
+      const tRow = createAndAppend('tr', tBody);
+      for (let j = 0; j < 2; j++) {
+        createAndAppend('td', tRow);
+        tRow.childNodes[0].innerText = headsOfDetails[i];
+      }
+    }
+    const tData = document.querySelector('tr').lastChild;
```

Comment:
I don't think you need to run the `querySelector` method on the `document` object. Since we are creating the `table` element above in this function, you can try to find the `tr` element inside it. It will save you some execution time as you won't be fetching the whole document but only a subset of it for the `tr` element. 🙂